docs/appendices/release-notes/5.10.4.rst

```diff
@@ -63,3 +63,5 @@ Fixes
   closed properly, e.g. due to an unclean node shutdown and the node was
   restarting with still holding a lock on the shard. The existing retry logic
   to solve this situation was not working in some cases.
+
+- Fixed an issue causing a CrateDB node to fail on startup in Windows environments when no custom CRATE_HEAP environment variable is set.
```

Comment:
I did not take care of max chars per line in my suggestion, sorry! Could you please fix this?